service/frontend/workflow_handler.go

```diff
@@ -6122,3 +6122,27 @@ func (wh *WorkflowHandler) DescribeWorker(ctx context.Context, request *workflow
 		WorkerInfo: resp.GetWorkerInfo(),
 	}, nil
 }
+
+// PauseWorkflowExecution pauses a workflow execution.
+func (wh *WorkflowHandler) PauseWorkflowExecution(ctx context.Context, request *workflowservice.PauseWorkflowExecutionRequest) (_ *workflowservice.PauseWorkflowExecutionResponse, retError error) {
```

Comment:
does pause workflow count as an action? if so we need to update the telemetry interceptor to emit an action metric with the api is called.